modules/manually-restoring-cluster-etcd-backup.adoc

```diff
@@ -0,0 +1,312 @@
+// * post_installation_configuration/cluster-tasks.adoc
+
+
+:_content-type: PROCEDURE
+[id="manually-restoring-cluster-etcd-backup_{context}"]
+= Restoring a cluster manually from an etcd backup
+
+You can use a saved etcd backup to restore a previous cluster state or restore a cluster that has lost the majority of control plane hosts. The procedure described below only uses the already existing control plane nodes.
+
+[NOTE]
+====
+The standard restore procedure requires the complete recreation of 2 control plane nodes, which might be complex for clusters installed with UPI. 
+====
+
+[NOTE]
+====
+If your cluster uses a control plane machine set, see "Troubleshooting the control plane machine set" for a more simple etcd recovery procedure.
+====
+
+[IMPORTANT]
+====
+When you restore your cluster, you must use an etcd backup that was taken from the same z-stream release. For example, an {product-title} 4.7.2 cluster must use an etcd backup that was taken from 4.7.2.
+====
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* A healthy control plane host to use as the recovery host.
+* SSH access to *all* control plane hosts, with the ability to switch to `root` user.
+* A backup directory containing both the etcd snapshot and the resources for the static pods, which were from the same backup. The file names in the directory must be in the following formats: `snapshot_<datetimestamp>.db` and `static_kuberesources_<datetimestamp>.tar.gz`.
+
+[IMPORTANT]
+====
+For non-recovery control plane nodes, it is not required to establish SSH connectivity or to stop the static pods. You can delete and recreate other non-recovery, control plane machines, one by one.
+====
+
+.Procedure
+
+. Select a control plane host to use as the recovery host. This is the host that you will run the restore operation on.
+
+. Establish SSH connectivity to each of the control plane nodes, including the recovery host.
++
+The Kubernetes API server becomes inaccessible after the restore process starts, so you cannot access the control plane nodes. For this reason, it is recommended to establish SSH connectivity to each control plane host in a separate terminal.
++
+[IMPORTANT]
+====
+If you do not complete this step, you will not be able to access the control plane hosts to complete the restore procedure, and you will be unable to recover your cluster from this state.
+====
+
+. Copy the etcd backup directory to the recovery control plane host.
++
+This procedure assumes that you copied the `backup` directory containing the etcd snapshot and the resources for the static pods to the `/home/core/` directory of your recovery control plane host.
+
+. Stop the static pods on all the control plane nodes, but one host at a time.
++
+[NOTE]
+====
+It is not required to manually stop the pods on the recovery host. The recovery script will stop the pods on the recovery host.
+====
+
+.. Access a control plane host that is not the recovery host.
```

Comment:
```suggestion
.. 
```

Reply:
Should be deleted, since Not manually stopping the static pods on the control plane node might mess with the etcd "regeneration" later, and therefore at this step al 3 control plane nodes are to be considered identical.

Reply:
Done.